<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorBalance</class>
 <widget class="QDialog" name="ColorBalance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>色彩平衡</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image.qrc">
    <normaloff>:/icon/color/Image/stock-color-color-balance.png</normaloff>:/icon/color/Image/stock-color-color-balance.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>要调整的范围</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_Shadow">
          <property name="text">
           <string>阴影</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_Midtone">
          <property name="text">
           <string>中间调</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_HighLight">
          <property name="text">
           <string>高光</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_preserve_luminosity">
        <property name="text">
         <string>保持亮度</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>调整颜色色阶</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>青色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_red">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>红色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_red">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>品红</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_green">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>绿色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_green">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>黄色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_blue">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>蓝色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_blue">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reset_range">
          <property name="text">
           <string>重置范围</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>ColorBalance</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ColorBalance</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
